### DISCUSSION TITLE
How can I know if any of the translations have been changed / updated?

### DISCUSSION THREAD

**Discussion by Manuel-Senpai** [Divested-Mobile/hypatia#26] [General]
Hi:
Sorry for the newbie question, I have never used "github" to translate....
I made the "Fork" some time ago and now I want to update my translation, but I have this doubt....
Once you have created a "Fork" If the developers change/add something in the main repository, 
is it also updated in the "Fork"?, 
how can I know if something was added/changed to the files I have to translate? 
in this case it would be the changes to the original file "README.md", and the original file "/app/src/main/res/values/strings.xml".

Best regards

**Comment by SkewedZeppelin** [comment]
No user facing strings have changed since your previous contribution.

>  is it also updated in the "Fork"?,

it is not, a fork is like a split in the road

you can update your fork like this if necessary:
```
git remote add upstream https://github.com/Divested-Mobile/Hypatia
git fetch upstream
git rebase upstream/stable
```
